app/models/court_data_api_resource.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# rubocop:disable Style/OptionalBooleanParameter
+
+module CourtDataApiResource
+  attr_accessor :active_record_fields_set
+
+  def connection(refresh = false)
+    set_my_active_record_fields unless active_record_fields_set
+    super(refresh)
+  end
+
+  def site
+    set_my_active_record_fields unless active_record_fields_set
+    super
+  end
+
+  def set_my_active_record_fields
+    self.site = Rails.configuration.x.court_data_api_config.uri
+  end
+end
+
```

Comment:
Not sure what the purpose of this code is, would you mind describing it?

Reply:
When we are testing against ActiveResource it has trouble pulling environment variables at the right time to set the site field and will through an exception. This Module allows for the site to be set in a timely fashion so that it will always work